lib/stats/stats-cluster.h

```diff
@@ -51,6 +51,8 @@ typedef enum _StatsClusterUnit
   SCU_NONE = 0,
 
   SCU_SECONDS,
+  /* represent value relative to the current time */
+  SCU_SECONDS_AGE,
```

Comment:
This struct represents the value we store, not the format we want to display.
I think we should handle this separately, by either introducing a "display format" on the formatter side, or we could just do the formatting based on the metric's name.

FYI year-2038 problem will happen with this one, for example, on armv7.

Reply:
hmm.. this is the same as the "stamp" counter for LogPipe instances. We also have SCU_SECONDS that uses the same 32 bit counter in this case.

But indeed due to the substraction in the formatting code, this would become a huge number after 2038. I see that @alltilla changed this code, let me comment on its newer iteration.

Reply:
I opened another discussion at the new code so closing this one.